### PR TITLE
Only retry Ro/Gy CCR if CCA RC is temporary error

### PR DESCRIPTION
**Pull request by fholzhauser**

### CODE REVIEW

**Review by vkatsuba — APPROVED** [review]

File: src/ergw_aaa_ro.erl [hunk 1]
```diff
@@ -227,15 +227,15 @@ handle_answer(#diameter_packet{msg = Msg, errors = Errors},
 	    [_ | #{'Result-Code' := RC}] -> RC;	%% try to handle gracefully
 	    _                            -> failed
 	end,
-    if Code >= 3000 ->
+    if Code >= 3000 andalso Code < 4000->
```
Comment (ebengt) [inline src/ergw_aaa_ro.erl:230]: Could it be that 3000 is not a code(*)? Then  you could write
if Code > 3000

(*)
https://www.developingsolutions.com/reference-guides/diameter_result_codes/

**Review by ebengt — APPROVED** [review]
Looks good

File: src/ergw_aaa_ro.erl (hunk 1 above)
Comment (RoadRunnr) [reply src/ergw_aaa_ro.erl:230]: 3000 is reserved

the authoritative source is IANA: https://www.iana.org/assignments/aaa-parameters/aaa-parameters.xhtml#aaa-parameters-7

**Review by RoadRunnr — APPROVED** [review]

File: src/ergw_aaa_ro.erl (hunk 1 above)
Comment (ebengt) [reply src/ergw_aaa_ro.erl:230]: When 3000 is reserved, can it appear in an answer?

**Review by RoadRunnr — APPROVED** [review]

**Pull request merged**